cmd/kubeadm/app/util/config/upgradeconfiguration.go

```diff
@@ -123,18 +108,8 @@ func LoadUpgradeConfigurationFromFile(cfgPath string, _ LoadOrDefaultConfigurati
 	// Convert documentMap to internal UpgradeConfiguration, InitConfiguration and ClusterConfiguration from config file will be ignored.
```

Comment:
the comments here can be updated to only say that everything other than UpgradeConfiguration will be ignored. instead of talking about the various kinds.

Reply:
updated